recipe/meta.yaml

```
@@ -11,19 +11,19 @@ source:
   sha256: 225e6b35252d68f7894716101fe3bd7e6dd3d30ab7b1c3c023f77a42dbe1336f
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  #[py<39]
```

https://github.com/lebrice/SimpleParsing/blob/v0.1.7/pyproject.toml#L10
python = ^3.8 -> python >=3.8,<4.0
```suggestion
  skip: True  #[py<38]
```